app/models/event.rb

```diff
@@ -2,7 +2,8 @@ class Event < ApplicationRecord
   belongs_to :created_by, class_name: "User", optional: true
   has_many :bookmarks, as: :bookmarkable, dependent: :destroy
   has_many :event_registrations, dependent: :destroy
-  
+  has_many :users, through: :event_registrations
```

Comment:
may we make this `has_many :registrants, through: :event_registrations, class_name: "User"` just so it's a little clearer?